src/index.html

```diff
@@ -60,4 +63,5 @@ <h1>2048</h1>
     </div>
     <script type="text/javascript" src="scripts/main.js"></script>
   </body>
+
 </html>
```

Comment:
```suggestion
  </body>
</html>
```
Don't leave empty lines between parent and child elements